DRCollectionViewTableLayout/DRCollectionViewTableLayoutManager.m

```diff
@@ -189,4 +189,23 @@ - (void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPa
     }
 }
 
+
+- (void)collectionView:(UICollectionView *)collectionView didDeselectItemAtIndexPath:(NSIndexPath *)indexPath {
+    NSUInteger row = [(DRCollectionViewTableLayout *)collectionView.collectionViewLayout rowNumberForIndexPath:indexPath];
+    NSUInteger column = [(DRCollectionViewTableLayout *)collectionView.collectionViewLayout columnNumberForIndexPath:indexPath];
+    
+    [self.delegate collectionViewTableLayoutManager:self
+                                     collectionView:collectionView
+                               didDeselectCellAtRow:row
+                                             column:column
+                                          indexPath:indexPath];
+}
+
+- (BOOL)collectionView:(UICollectionView *)collectionView shouldSelectItemAtIndexPath:(NSIndexPath *)indexPath {
+    NSUInteger row = [(DRCollectionViewTableLayout *)collectionView.collectionViewLayout rowNumberForIndexPath:indexPath];
+    NSUInteger column = [(DRCollectionViewTableLayout *)collectionView.collectionViewLayout columnNumberForIndexPath:indexPath];
+    
+    return [self.delegate collectionViewTableLayoutManager:self collectionView:collectionView shouldSelectItemAtRow:row column:column indexPath:indexPath];
```

Comment:
Because  `collectionViewTableLayoutManager:collectionView:shouldSelectItemAtRow:column:indexPath:` method is defined as optional, delegate don't have to implement it. Before calling this method on delegate, check if it's implemented using `respondsToSelector:` method.